ts/features/payments/bizEventsTransaction/components/PaymentsBizEventsTransactionCartList.tsx

```diff
@@ -38,11 +39,14 @@ export const PaymentsBizEventsTransactionCartList = ({
           key={cartItem.refNumberValue ?? index.toString()}
           title={cartItem.subject ?? ""}
           subtitle={cartItem.payee?.name ?? ""}
-          transactionStatus="success"
-          transactionAmount={
-            cartItem.amount ? formatAmountText(cartItem.amount) : ""
-          }
-          hasChevronRight
+          transaction={{
+            amount: cartItem.amount ? formatAmountText(cartItem.amount) : "",
```

Comment:
What do you think to assign it like this?

```
amount: formatAmountText(cartItem.amount ?? "")
```

Check `formatAmountText` function. It will return an empty string if the given cannot be parsed as float number

Reply:
Addressed into edd87d2dd775d47e732c551dcb08eb86089fb4d9